sdk/azcore/request.go

```diff
@@ -144,6 +144,13 @@ func (req *Request) SkipBodyDownload() {
 	req.SetOperationValue(bodyDownloadPolicyOpValues{skip: true})
 }
 
+// returns true if auto-body download policy is enabled
+func (req *Request) bodyDownloadEnabled() bool {
+	var opValues bodyDownloadPolicyOpValues
+	req.OperationValue(&opValues)
```

Comment:
What is this line doing?

Reply:
The request object contains a property bag `values` where you can stash various things.  This line is retrieving it from the property bag if available and populating the value.

Reply:
Ah I see. Ok cool. Thanks for the explanation! 👍